### PR TITLE
Add weekly update 2015-12-11

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2015-12-11.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+See https://nodejs.org/en/blog/announcements/apigee-rising-stack-yahoo/ for more information.
+
+### Node.js Foundation Advances Platform with More Than Three Million Users
+
+* [The Node.js Foundation](https://nodejs.org/en/foundation/), announced major community, code and membership growth, adoption statistics of the technology at large, and the Foundation’s new incubation program. 
+* Since the independent Node.js Foundation launched earlier this year, development progress continues to accelerate with dramatic increases in contributions to the project.
+* The first Node.js Interactive event unites more than 700 developers, engineers, system architects, DevOps professionals and users representing a wide range of projects, products and companies in Portland, Ore. 
+* Node.js Interactive brings together a broad range of speakers to help experienced and novice Node.js users alike learn tips, best practices, new skills, as well as gain insight into future developments for the technology.
+* The Node.js Foundation announced its first incubation project libuv
```
Comment (ghost) [inline locale/en/blog/weekly-updates/weekly-update.2015-12-11.md]: minor nitpick - missing `.` at the end?

File: locale/en/blog/weekly-updates/weekly-update.2015-12-11.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* The project is both critical to Node.js and already widely used, making it a natural fit for the Foundation.
+* Under the Foundation's umbrella, it will receive additional support and mentorship.
+
+See https://nodejs.org/en/blog/announcements/foundation-advances-growth/ for more information.
+
+### Community Updates
+
+* [Finding a Memory Leak in Node.js](https://blog.risingstack.com/finding-a-memory-leak-in-node-js/), "We compiled a bunch of methods and tools that could help"
+* [Node.js running in the new Airbus A350 inflight servers](http://reaktor.com/blog/aircraft-customer-experience-on-a-new-level/), "We were allowed to run our own node.js-server on the inflight servers of the aircraft."
```
Comment (ghost) [inline locale/en/blog/weekly-updates/weekly-update.2015-12-11.md]: capitalize `node.js` for uniformity :smile:

File: locale/en/blog/weekly-updates/weekly-update.2015-12-11.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+* [Automatic cat feeder powered by Node.js](https://github.com/rachelnicole/robokitty), "Is a DIY cat (or dog, or human) feeder controlled over the web."
```
Comment (ghost) [inline locale/en/blog/weekly-updates/weekly-update.2015-12-11.md]: `Is a DIY [...]` sounds wrong, how about `A DIY [...]` or something?

**Comment by JungMinu** [comment]
@sup Thanks, revised and updated :)

**Pull request merged**